NEWS

```diff
@@ -18,6 +18,20 @@ CHANGES WITH 253 in spe:
           For more details, see:
           https://lists.freedesktop.org/archives/systemd-devel/2022-September/048352.html
 
+        * We intend to change behaviour w.r.t. user units and sandboxing
```

Comment:
user units is a big vague, people might understand that as units tha thave User= set.

explicitly say "units of the per-user service manager" or so.